.github/workflows/build-test.yml

```diff
@@ -85,14 +85,13 @@ jobs:
         uses: OffchainLabs/actions/run-nitro-test-node@main
         with:
           nitro-testnode-ref: release
-          # don't deploy the regular token bridge using arbitrum-sdk, as it will be deployed through the TokenBridgeCreator
-          no-token-bridge: true
+          # Use simple mode
+          no-simple: false
```

Comment:
Since the testnode action changed, some flags have been adjusted (no need to use --no-tokenbridge anymore, and we can activate simple mode for speed)